src/containers/treePage/index.tsx

```diff
@@ -149,62 +182,99 @@ const TreePage: React.FC<SiteProps> = ({ address, entries }) => {
         />
       </Helmet>
       <PageLayout>
-        <TreePageContainer>
-          {/*Change to tree map route once that page is finished*/}
-          <ReturnButton to={Routes.HOME}>{`<`} Return to Tree Map</ReturnButton>
-          <TreeMainContainer>
+        {asyncRequestIsComplete(siteData) && (
+          <TreePageContainer>
+            {/*Change to tree map route once that page is finished*/}
```

Comment:
Right now this is just the landing, do you think we should have a separate page for a tree map too?

Reply:
No, moving this to the landing is fine